_drafts/oryx/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+Routing is an essential feature for any web application, allowing users to navigate between different pages or components within the application. The `@spryker-oryx/router` package
+provides routing infrastructure for your application, including a `RouterService` for interacting with the router and framework-specific integrations for providing actual routing config.
+This guide will walk you through the process of setting up routing in your application, including how to add the `RouterFeature` to your application, render the router outlet, and provide routes using Dependency Injection (DI).
+
+`@spryker-oryx/router` provides infrastructure for routing capabilities. It gives you mainly the `RouterService` which is a generic way for navigating in your application.
```

Comment:
It's the same info as in the first paragraph. Where would it make more sense?

Reply:
I guess we can remove the second sentence from the last paragraph to reduce duplicated info.

Reply:
I've rearranged it and removed it from the first paragraph.